Modix.Services/UserInfo/UserInfoEmbedBuilderHelper.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Discord;
+using Humanizer;
+using Modix.Common.Extensions;
+using Modix.Data.Models.Moderation;
+using Modix.Services.Extensions;
+using Modix.Services.Utilities;
+
+namespace Modix.Services.UserInfo
+{
+    public class UserInfoEmbedBuilderHelper
+    {
+        private readonly StringBuilder _content = new StringBuilder();
+        private readonly DateTimeOffset _nowUtc = DateTimeOffset.UtcNow;
+
+        public static UserInfoEmbedBuilderHelper Create()
+        {
+            return new UserInfoEmbedBuilderHelper();
+        }
+
+        private UserInfoEmbedBuilderHelper()
+        {
+            _content.AppendLine("**\u276F User Information**");
+        }
+
+        public UserInfoEmbedBuilderHelper WithId(ulong userId)
+        {
+            _content.AppendLine("ID: " + userId);
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithClickableMention(ulong userId)
+        {
+            _content.AppendLine("Profile: " + MentionUtils.MentionUser(userId));
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithFirstAndLastSeen(DateTimeOffset firstSeen, DateTimeOffset lastSeen)
+        {
+            _content.AppendLine($"First Seen: {FormatUtilities.FormatTimeAgo(_nowUtc, firstSeen)}");
+            _content.AppendLine($"Last Seen: {FormatUtilities.FormatTimeAgo(_nowUtc, lastSeen)}");
+
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithStatus(UserStatus status)
+        {
+            _content.AppendLine($"Status: {status.Humanize()}");
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithBan(string reason)
+        {
+            _content.AppendLine($"🔨\\ **Banned**: {reason}");
```

Comment:
```suggestion
            _content.AppendLine($"\\🔨 **Banned**: {reason}");
```
The slash in front of the emoji was so that it renders as the smaller unicode version instead of the larger twemoji version

![image](https://user-images.githubusercontent.com/2829282/73126050-de389980-3f7b-11ea-9e08-cb8a20897b98.png)